source/install/desktop-app-changelog.rst

```diff
@@ -6,19 +6,70 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.4 <#id1>`_
-- `Release v5.3 <#id3>`_
-- `Release v5.2 <#id16>`_
-- `Release v5.1 <#id32>`_
-- `Release v5.0 <#id47>`_
-- `Release v4.7 <#id62>`_
+- `Release v5.5 <#id1>`_
+- `Release v5.4 <#id3>`_
+- `Release v5.3 <#id15>`_
+- `Release v5.2 <#id28>`_
+- `Release v5.1 <#id44>`_
+- `Release v5.0 <#id59>`_
+
+Release v5.5
+--------------
+
+**Release Day: September 15, 2023**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Note:** Mattermost v5.5.0 contains a medium severity level security fix. Upgrading is highly recommended. Details will be posted on our `security updates page <https://mattermost.com/security-updates/>`__ 30 days after release as per the `Mattermost Responsible Disclosure Policy <https://mattermost.com/security-vulnerability-report//>`__.
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
+- Updated Chromium minimum supported version to 116+.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Set the minimum window width to 600px.
+
+Architectural Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Major version upgrade of Electron to v26.1.0. Electron is the underlying technology used to build the Desktop App.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Fixed a crash in diagnostics when the server was unreachable.
+- Fixed bad user feedback on the server URL validation when plugins were disabled.
+- Fixed an issue where auto-updating the app wouldn't be properly disabled.
+
+Known Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Users seeing an endless "Loading..." screen when attempting to log in to the app may need to manually remove their cache directory. For MacOS it is located in ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost`` and for Windows it is located in ``Users/<username>/AppData/Roaming/Mattermost``.
```

Comment:
```suggestion
- Users seeing an endless "Loading..." screen when attempting to log in to the app may need to manually delete their cache directory. For MacOS it is located in ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost`` and for Windows it is located in ``Users/<username>/AppData/Roaming/Mattermost``.
```

Reply:
Can we also include the path for Linux OS users?

Reply:
Maybe @devinbinnie can help with that.

Reply:
For Linux it should be `~/config/Mattermost`, where ~ is the home directory.